test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -299,3 +299,24 @@ Describe "Connection" -Tag "CI", "RequireAdminOnWindows" {
         Test-Connection $UnreachableAddress -TcpPort 80 -TimeOut 1 | Should -BeFalse
     }
 }
+
+Describe "Test-Connection should run in the default synchronization context (threadpool)" -Tag "CI" {
+    It "Test-Connection works after constructing a WindowsForm object" -Skip:(!$IsWindows) {
+        $pwsh = Join-Path $PSHOME "pwsh"
+        $pingResults = & $pwsh -noprofile {
```

Comment:
```suggestion
        $pingResults = & $pwsh -NoProfile -NonInteractive {
```

Reply:
I will update as suggested, but is there a reason why we need `-NoInteractive`?